### ISSUE TITLE
Wave/WCAG errors when setting descriptions on components shown in `Grid` 

### ISSUE THREAD

**Issue by olemartinorg** [Altinn/app-frontend-react#1629]
### Description of the bug

When showing for example an Input field in a Grid component (or any other table, presumably), the component renders with a broken `aria-describedby` reference, as the description is not rendered in the table. We should probably either render the description invisibly when in a table, or figure out a solution that keeps accessibility promises.

### Steps To Reproduce

1. Open the `ra0483-01` app as in the example linked below
2. Navigate to the second page that contains a `Grid`
3. Answer yes to the question so the `Grid` appears
4. Run Wave

### Additional Information

- https://altinn.slack.com/archives/C02EVE4RU82/p1697630977852459
